test/metabase/query_processor_test/timezones_test.clj

```diff
@@ -230,6 +242,16 @@
    (when (supports-datetime-with-zone-id?)
      {:datetime_tz_id (t/zoned-date-time "2019-11-01T00:23:18.331-07:00[America/Los_Angeles]")})))
 
+(deftest sql-datetime-timezone-handling-test
+  (mt/test-drivers (filter #(isa? driver/hierarchy % :sql) (conj (set-timezone-drivers) :oracle)) ; Oracle doesn't have a time type, so it's excluded from this test
+    (mt/dataset dt-attempted-murders
+      (doseq [timezone ["UTC" "US/Pacific" "US/Eastern" "Asia/Hong_Kong"]]
+        (mt/with-temporary-setting-values [report-timezone timezone]
```

Comment:
Why to set report timezone here if `:format-rows?` is set to false? My understanding is that report timezone is set as session timezone, results for ltz are returned in that, but zone offset is converted to 0 in read-column-thunk. Hence actual query results are unaffected.

Reply:
My reasoning: `:format-rows?` is set to false in `sql-time-timezone-handling-test`. So `sql-time-timezone-handling-test` is testing a property of the query processor before row formatting is applied. So we should do the same thing here with `:datetime_ltz`.

It's potentially flimsy reasoning and worth questioning, and I'm openly admitting I don't understand all the guarantees the QP is meant to make. They're not all enforced by tests currently. 

But I believe that because `sql-time-timezone-handling-test` tests that values before row formatting have offsets of 0 independent of `report-timezone`, we should do the same in this test for consistency at least.